common/logger_utility.py

```diff
@@ -1,37 +1,38 @@
 import logging
 import os
-from common.constants import *
+
+from common.constants import Constants
 
 
 class LoggerUtility:
     @staticmethod
-    def setLevel():
-        logFormat = '%(asctime)-15s %(levelname)s:%(message)s'
-        logging.basicConfig(format=logFormat)
+    def set_level():
```

Comment:
Could we run pylint in the build? This way we can assured that everyone is checking in stylistically correct python code. It will prevent us from having to have someone make changes again like this in this lambda...